### PR TITLE
Update dependencies

### LINKED ISSUE #5711: Update dependencies
Need to update these dependencies on priority:

- marshmallow = ">=2.15.1"
- requests = ">=2.20.0"

### PR DESCRIPTION
**Pull request by pnijhara**
Fixes #5711 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

Updated two dependencies:
1.    marshmallow = ">=2.15.1"
2.    requests = ">=2.20.0"

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Make changes in requirements.txt

File: Pipfile.lock [hunk 1]
```diff
@@ -505,7 +505,7 @@
                 "sha256:5f7a9ab5d5f5583fe4e071fde98a472b04fd0df868ead83012f1a3e0e54d7cdf",
                 "sha256:cf7e35ddfe8b18f8de3058fffc621921c79bb07455ad3084bfbf64caca766711"
             ],
-            "version": "==2.13.1"
+            "version": ">=2.15.1"
```
Comment (iamareebjamal) [inline Pipfile.lock]: This file should not be changed manually

**Comment by pnijhara** [comment]
@iamareebjamal Marshmallow was not in requirements.txt

**Comment by iamareebjamal** [comment]
It must be a transitive dependency, add it

**Comment by iamareebjamal** [comment]
Where's the change in requests

**Comment by pnijhara** [comment]
Should I make a change in requests[security]>=2.12.4,<3  or add requests = ">=2.20.0" separately?

**Comment by iamareebjamal** [comment]
Change. And why removed the marchmallow change from Pipfile?

**Comment by iamareebjamal** [comment]
Also, sync the Pipfile.lock after changes

**Comment by pnijhara** [comment]
> Change. And why removed the marchmallow change from Pipfile?

There was no marshmallow in pipflie. There was only requests.

**Comment by iamareebjamal** [comment]
Similarly add marshmallow to Pipfile and sync Pipfile.lock

File: Pipfile [hunk 2]
```diff
@@ -61,6 +61,7 @@ factory_boy = "*"
 Flask-Redis = "*"
 SQLAlchemy = "==1.1.15"
 Flask-Elasticsearch = "*"
+marshmallow = ">=2.15.1"
```
Comment (iamareebjamal) [inline Pipfile]: ,<3

File: requirements/common.txt [hunk 3]
```diff
@@ -56,3 +56,4 @@ Flask-Elasticsearch
 paypalrestsdk
 eventlet
 pyyaml~=4.2b1
+marshmallow>=2.15.1
```
Comment (iamareebjamal) [inline requirements/common.txt]: ,<3

**Comment by pnijhara** [comment]
@iamareebjamal please review it.

**Comment by iamareebjamal** [comment]
I have already. See requested change

**Comment by pnijhara** [comment]
@iamareebjamal Done with the requested changes in the previous commit. Do let me know if any further changes require.

**Comment by iamareebjamal** [comment]
Required changes still not met

Mentioned once here https://github.com/fossasia/open-event-server/pull/5714#issuecomment-476690178

Twice here: https://github.com/fossasia/open-event-server/pull/5714#issuecomment-476788922

Thrice here: https://github.com/fossasia/open-event-server/pull/5714#pullrequestreview-219103708

Still not done

**Pull request closed without merging**